dla/dla.md

```diff
@@ -0,0 +1,12 @@
+# FirstName LastName (Unique ID): dla
+## Profile Pic: 
+## Call me by this name: daekyung
+## Cohort: July 2023
+## Email ID: lakoda@kookmin.ac.kr
+## LinkedIn: 
+## Slack ID: Daekyung La
+## GitHub ID: Ladaekyung
+## About me: KMU students
```

Comment:
May be something more interesting than just "KMU students" ? :smiling_face_with_tear: